### ISSUE TITLE
Implement a central repository for packages

### ISSUE THREAD

**Issue by kivimango** [nim-lang/packages#1039]
Hello.
This package list is very basic and after some time it will be unmaintenable if the ecosystem grows bigger.
I can implement a REST API for storing and retrieving nim packages in my free time if you are interested.

**Comment by genotrance** [comment]
Also see:

https://github.com/nim-lang/packages/issues/1012
https://github.com/nim-lang/packages/issues/777
http://nimble.directory

**Comment by kivimango** [comment]
https://github.com/kivimango/nim-hub

**Comment by dom96** [comment]
Written in Java? Come onnnnn

**Comment by kivimango** [comment]
Yes.Java has a richer ecosystem, and some of the libs (securtiy, testing) i couldnt find written in nim.
One could rewrite this later in nim ofc.

**Comment by dom96** [comment]
Do what you like, but there is a low chance we would adopt a Nim "hub" written in Python, let alone Java.

**Comment by FedericoCeratto** [comment]
See https://github.com/nim-lang/packages/issues/777#issuecomment-562863060
